hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/systemcontracts/PrngSystemContractTest.java

```diff
@@ -101,17 +116,16 @@ void onlyFullResultsAreSupported() {
     @Test
     void computePrecompileStaticSuccessTest() {
         // given:
-        givenCommonBlockValues();
+        givenInitialFrame();
         given(messageFrame.isStatic()).willReturn(true);
         given(messageFrame.getWorldUpdater()).willReturn(proxyWorldUpdater);
         given(proxyWorldUpdater.entropy()).willReturn(EXPECTED_RANDOM_NUMBER);
 
         // when:
-        var actual = subject.computeFully(PSEUDO_RANDOM_SYSTEM_CONTRACT_ADDRESS, messageFrame)
-                .result();
+        var actual = subject.computeFully(PSEUDO_RANDOM_SYSTEM_CONTRACT_ADDRESS, messageFrame);
 
         // then:
-        assertEqualContractResult(PRECOMPILE_CONTRACT_SUCCESS_RESULT, actual);
+        assertEqualContractResult(PRECOMPILE_CONTRACT_SUCCESS_RESULT, actual, 100L);
```

Comment:
You gave `GAS_REQUIRED` a name, why not this `100L`?  And speaking of `GAS_REQUIRED` - from **my personal POV** it wasn't really a good name as I wasn't sure whether this was gas required by any pnrg call, or this pnrg call, or what.  Suggestion would be to make this a bit more explicit to help lame readers like me?